plugins/rbac-backend/src/database/conditional.storage.test.ts

```diff
@@ -83,7 +83,7 @@ describe('DataBaseConditionalStorage', () => {
 
   async function createDatabase(databaseId: TestDatabaseId) {
     const knex = await databases.init(databaseId);
-    const databaseManagerMock: PluginDatabaseManager = {
+    const databaseManagerMock: DatabaseService = {
```

Comment:
You can refer to the OCM router tests [here](https://github.com/janus-idp/backstage-plugins/pull/2104/files#diff-4327a4f681596c8e0c0869c6cea617aa42ce2226867653ed0190ee0093051254). Going forward, we should avoid manually mocking any services.